### DISCUSSION TITLE
[Question] How can we allow touch outside, when selector is opened?

### DISCUSSION THREAD

**Discussion by OvsTim** [MobileReality/react-native-select-pro#130] [Q&A] [answered]
So i use this libriary in my app, when user selects place on map, dropdown with places appears.

But when i want to select another place, i should tap anywhere on screen to dismiss selector and then i can use map again.

Did anybody khow, what should i do? its possible at all?

**Comment by irekrog** [comment, marked as answer]
Hi @OvsTim 

At this moment you can't do this with any prop. But you can try to use a `patch-package` lib and comment out this block of code ([242-252](https://github.com/MobileReality/react-native-select-pro/blob/master/packages/react-native-select-pro/src/components/options-list/index.tsx#L242))  But I can't guarantee that the component will still work well after that

**Comment by sizuf** [comment]
As of 2024 what we have to do in order to make that work? I.e the component goes out of focus when pressed outside?
